openslides_backend/action/util/assert_belongs_to_meeting.py

```diff
@@ -26,6 +26,12 @@ def assert_belongs_to_meeting(
             )
             if meeting_id not in instance.get("meeting_ids", []):
                 errors.add(str(fqid))
+        elif fqid.collection.collection == "mediafile":
+            mediafile = datastore.get(fqid, ["owner_id"])
+            collection, id_ = mediafile["owner_id"].split(KEYSEPARATOR)
+            if collection == "meeting":
```

Comment:
if the collection is not meeting, an error must also be thrown, since it does not belong to the meeting

Reply:
If the collection is not a meeting collection, throw an error.